charts/policies/allow-privilege-escalation-psp/values.yaml

```diff
@@ -0,0 +1,24 @@
+---
+kind: ClusterAdmissionPolicy
+metadata:
+  name: allow-privilege-escalation-psp
+  namespace: default # used when kind is AdmissionPolicy
+spec:
+  module:
+    repository: "registry://ghcr.io/kubewarden/policies/allow-privilege-escalation-psp"
```

Comment:
Note that this `repository`:`tag` corresponds to an OCI artifact, not a container image. So it will need to be pulled from/pushed to an OCI registry.
Right now, [`spec.module` expects a prefix of either `registry://` (oci registry), `file://`, `http://`, `https://`](https://doc.crds.dev/github.com/kubewarden/kubewarden-controller/policies.kubewarden.io/AdmissionPolicy/v1@v1.0.0#spec-module).

I suppose that Rancher should first use an OCI registry, and then, the script that compiles the list of images for airgap should support the usual case, which is `registry://`.

We could also separate the prefix in `values.yaml`, having:
```
spec:
  module:
    prefix: "registry://"
    repository: "ghcr.io/kubewarden/policies/allow-privilege-escalation-psp"
    tag: "v1.1.7"
```

But the tool should then recognize `spec.module.prefix` and skip the images if it's `file://` for example.

Another option is to accept `.spec.module.repository` without a protocol prefix, and assume internally that it is `registry://`. This would need to change to the airgap scripts, but still needs a Rancher OCI registry.

Reply:
Removed unneeded `registry://` prefix in https://github.com/kubewarden/helm-charts/pull/131/commits/7d53ae0fe98053e933825759c00d4a54b6b2bee5.